src/App.tsx

```diff
@@ -1,8 +1,24 @@
 import React from "react";
+import { ApolloClient, InMemoryCache, ApolloProvider } from "@apollo/client";
 import "./styles.scss";
+import { BrowserRouter, Routes, Route } from "react-router-dom";
+
+import CharacterDetails from "./pages/CharacterDetails";
+const client = new ApolloClient({
+  uri: "https://rickandmortyapi.com/graphql",
```

Comment:
you can get the uri from env file, ideally

Reply:
ok then, I will create one. Is this something related to making the app reusable?

Reply:
Sort of. Read abt it [here](https://medium.com/chingu/an-introduction-to-environment-variables-and-how-to-use-them-f602f66d15fa#:~:text=The%20primary%20use%20case%20for,to%20changes%20in%20configuration%20data.)